src/lib/journey/_utilities/step.utilities.ts

```diff
@@ -55,6 +56,19 @@ export function initCheckValidation() {
   };
 }
 
+/**
+ * @function shouldRedirectFromStep -
+ * @returns {boolean}
+ */
+export function shouldRedirectFromStep(step: FRStep) {
+  try {
+    step.getCallbackOfType(CallbackType.RedirectCallback);
+    return true;
+  } catch (e) {
+    return false;
+  }
```

Comment:
Rather than doing a `try-catch` with `getCallbackOfType` (which throws when the cb doesn't exist), I'd use `getCallbacksOfType` (which returns an array regardless of cb existence) and check the length of the array for what value to return.

Reply:
I see what you're saying, but I found getCallbacksOfType threw an error when there were no redirect callbacks. I'll try that again to be sure though.

Reply:
Huh, it shouldn't, but if it does, we should make sure there isn't a regression. Here's the source for this method: https://github.com/ForgeRock/forgerock-javascript-sdk/blob/develop/packages/javascript-sdk/src/fr-auth/fr-step.ts#L61.

Reply:
I will do more testing on it tomorrow, but it seems to throw an error - for example the above refactored to this:

```
if (step.getCallbacksOfType(CallbackType.RedirectCallback)) return true;
return false;
```

.. throws the following error in the console:

```
Uncaught (in promise) Error: Expected 1 callback of type "RedirectCallback", but found 0
    getCallbackOfType fr-step.ts:50
```

which doesn't make sense to me since it seems to be `getCallbackOfType()` that is failing despite calling `getCallbacksOfType`?

Reply:
I'll investigate further separate to this PR - @cerebrl I don't think this is blocking?

Reply:
Let's revisit this in the future, but it's non-blocking at this point.